tests/cpp/signal_processing_demo/src/signal_filtering.cpp

```diff
@@ -21,7 +21,7 @@ int main (int argc, char *argv[])
     int board_id = (int)BoardIds::SYNTHETIC_BOARD;
 
     BoardShim::enable_dev_board_logger ();
-    DataFilter::enable_dev_board_logger ();
+    DataFilter::enable_dev_board_logger();
```

Comment:
lets keep code style, you can enable clang-format and it will format all your code automatically on save(https://brainflow.readthedocs.io/en/stable/BrainFlowDev.html#code-style) I use sublime with clang-format plugin for example